patches/lottie-react-native+6.3.1.patch

```diff
@@ -0,0 +1,73 @@
+diff --git a/node_modules/lottie-react-native/src/LottieView.tsx b/node_modules/lottie-react-native/src/LottieView.tsx
+index 6372ca2..c5cfad7 100644
+--- a/node_modules/lottie-react-native/src/LottieView.tsx
++++ b/node_modules/lottie-react-native/src/LottieView.tsx
+@@ -4,6 +4,7 @@ import {
+   NativeSyntheticEvent,
+   ViewProps,
+   processColor,
++  StyleSheet,
+ } from 'react-native';
+ 
+ import type { LottieViewProps } from './LottieView.types';
+@@ -12,7 +13,7 @@ import NativeLottieAnimationView, {
+   Commands,
+ } from './specs/LottieAnimationViewNativeComponent';
+ 
+-type Props = LottieViewProps & { containerProps?: ViewProps };
++type Props = LottieViewProps & { containerProps?: ViewProps; autoSize?: boolean };
+ 
+ const defaultProps: Props = {
+   source: undefined,
+@@ -27,6 +28,7 @@ const defaultProps: Props = {
+   colorFilters: [],
+   textFiltersAndroid: [],
+   textFiltersIOS: [],
++  autoSize: false,
+ };
+ 
+ export class LottieView extends React.PureComponent<Props, {}> {
+@@ -132,6 +134,7 @@ export class LottieView extends React.PureComponent<Props, {}> {
+       textFiltersAndroid,
+       textFiltersIOS,
+       resizeMode,
++      autoSize,
+       ...rest
+     } = this.props;
+ 
+@@ -149,6 +152,22 @@ export class LottieView extends React.PureComponent<Props, {}> {
+       color: processColor(colorFilter.color),
+     }));
+ 
++    const aspectRatioStyle = sources.sourceJson
++      ? { aspectRatio: (source as any).w / (source as any).h }
++      : undefined;
++
++    const styleObject = StyleSheet.flatten(style);
++    let sizeStyle;
++    if (
++      !styleObject ||
++      (styleObject.width === undefined && styleObject.height === undefined)
++    ) {
++      sizeStyle =
++        autoSize && sources.sourceJson
++          ? { width: (source as any).w }
++          : StyleSheet.absoluteFill;
++    }
```

Comment:
I don't think that we want to simply revert [the breaking changes](https://github.com/lottie-react-native/lottie-react-native/pull/992/files#diff-989a0a34ad1a1c41bcd903ca888ab92e813b132c99db023f860eb6bddc7a736a) in the patch.

Reply:
Can we override using props? I don't really like patching

Reply:
We might also consider passing `aspectRatio` as a value/prop instead of calculating it. Just an idea.

Reply:
We should probably also put it into some `useMemo`. I think that this is still much better than doing it in a patch. Especially when I am working on [this related issue](https://github.com/Expensify/App/issues/26871). And patching would require regular maintenance if the original library changes.

---

But **there is a problem** with this approach. When we move to [the `.lottie`](https://github.com/Expensify/App/issues/26871) files we can't access the `.w` and `.h` so easily, as these animations won't be JSON objects anymore. I'm not sure how should we handle it.

---

cc @situchan @roryabraham  what do you think?